pina/callbacks/adaptive_refinment_callbacks.py

```diff
@@ -96,10 +96,12 @@ def _r3_routine(self, trainer):
         avg = (tot_loss.mean()).to("cpu")
 
         # points to keep
-        old_pts = {}
-        tot_points = 0
+        old_pts = {} # points to retain
+        left_pts = {} # number of points to be uniformly sampled from the respective location
         for location in self._sampling_locations:
+            tot_points = 0
```

Comment:
is this variable used?

Reply:
`tot_points` count the number of retained points, used to compute the number of `left_pts`. I will remove it along with the removal of `left__pts`.